dry_run/tasks/simple.py

```diff
@@ -0,0 +1,49 @@
+from cloudferrylib.scheduler import task
+from cloudferrylib.utils import utils as utl
```

Comment:
Very minor, but what is the reason of saving 2 bytes here? :) Just leave it as `from cloudferrylib.utils import utils as utl`, "utils" is much more readable than "utl".